### PR TITLE
refactor: refactor config, update readme, manifest

### PR DESCRIPTION
**Pull request by KostaZx**
## Description

Refactored config, updated readme, manifest.

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

**Comment by bicatcho11** [comment]
You have submitted pull requests
I hope to work sequentially from old to new

File: config/shared.lua [hunk 1]
```diff
@@ -0,0 +1,98 @@
+return {
+    useTarget = true, -- Use ox_target interactions (don't change this, go to your server.cfg and add `setr UseTarget true` to use this and just that from true to false or the other way around)
```
Comment (solareon) [inline config/shared.lua]:
```suggestion
    useTarget = true, -- Use ox_target interactions
```

File: fxmanifest.lua [hunk 2]
```diff
@@ -10,7 +10,7 @@ ox_lib 'locale'
 shared_scripts {
     '@ox_lib/init.lua',
     '@qbx_core/modules/lib.lua',
-    'config.lua'
+    'config/shared.lua'
```
Comment (solareon) [inline fxmanifest.lua]: This should be in the files section not as a shared script as we don't need to execute it

File: client/prisonbreak.lua
Comment (solareon) [inline client/prisonbreak.lua:1]: I believe that his has not been fully converted.

**Review by solareon — CHANGES_REQUESTED** [review]
Please address the linting errors and the other comments.

**Review by Manason — APPROVED** [review]

**Pull request merged**